Tasks/UsePythonVersionV0/task.json

```diff
@@ -30,8 +30,8 @@
             "type": "string",
             "label": "Version spec",
             "required": true,
-            "defaultValue": ">= 3.3",
-            "helpMarkDown": "Version range or exact version of a Python version to use."
+            "defaultValue": "3.x",
```

Comment:
Synced with @DavidStaheli on this, we think this is cleaner.  The semantics are essentially the same as >= 3.3 (Pythons before 3.3 are very old and were essentially beta releases of Python 3.  3.4 is the oldest version we are currently shipping on an agent.)

Reply:
+1